cms/config/database.ts

```diff
@@ -63,7 +63,7 @@ export default ({ env }) => {
           cipher: env('DATABASE_SSL_CIPHER', undefined),
           rejectUnauthorized: env.bool(
             'DATABASE_SSL_REJECT_UNAUTHORIZED',
-            true
+            false // I had to set this for production
```

Comment:
this smells

Reply:
I know, that is because I don't know how to separate the dev and prod environment for the db :/ was going to ask Inga about this